doc/install.rst

```diff
@@ -16,15 +16,28 @@ Stable Release
 Python
 ------
 
-Pre-built binary are uploaded to PyPI (Python Package Index) for each release.  Supported platforms are Linux (x86_64, aarch64), Windows (x86_64) and MacOS (x86_64, Apple Silicon).
+Pre-built binary wheels are uploaded to PyPI (Python Package Index) for each release. Supported platforms are Linux (x86_64, aarch64), Windows (x86_64) and MacOS (x86_64, Apple Silicon).
 
 .. code-block:: bash
 
+  # Pip 21.3+ is required
```

Comment:
XGBoost requires an in-tree build, which became the default since Pip 21.3+.

From Pip's changelog:

> 21.3 (2021-10-11)
> Deprecations and Removals
> * In-tree builds are now the default. --use-feature=in-tree-build is now ignored. --use-deprecated=out-of-tree-build may be used temporarily to ease the transition. ([#10495](https://github.com/pypa/pip/issues/10495))

Reply:
Got it. Looks like it can also be done optionally as of 21.1, but no need to change the requirement for that reason.